src/main/java/picard/vcf/VcfToIntervalList.java

```diff
@@ -84,10 +91,23 @@ protected int doWork() {
         IOUtil.assertFileIsReadable(INPUT);
         IOUtil.assertFileIsWritable(OUTPUT);
 
-        final IntervalList intervalList = VCFFileReader.fromVcf(INPUT, INCLUDE_FILTERED);
+        try (VCFFileReader vcfReader = new VCFFileReader(INPUT.toPath(), false)) {
+            final Iterator<Interval> samFileIterator = VCFFileReader.toIntervals(vcfReader, INCLUDE_FILTERED);
+            try (IntervalListWriter writer = new IntervalListWriter(OUTPUT.toPath(), new SAMFileHeader(vcfReader.getFileHeader().getSequenceDictionary()))) {
+                final IntervalList.IntervalMergerIterator mergingIterator =
+                        new IntervalList.IntervalMergerIterator(samFileIterator, true, false, true);
```

Comment:
Would be nice to expose to the command line the option to not concatenate all the merged interval names.  In the use case which prompted this, a TON of sites (think whole genome gvcf with bp resolution) are often merged into a single interval, which then has an outrageously long concatenated name.  These concatenated names often don't carry any useful information, and can make the resulting interval list extremely large.

Reply:
Yeah, 👍

Reply:
like so?

Reply:
yep!

Reply:
@skwalker can you try this on your VCF that was causing all the trouble and see if it helps?

Reply:
It works! Thank you (the output interval list went from almost 10G to less than 1 with that change)